.github/workflows/pr-check.yaml

```diff
@@ -11,9 +11,9 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '20'
       - name: Install AsyncAPI Generator
-        run: npm install -g @asyncapi/generator
+        run: npm install -g @asyncapi/generator@1.17.25
```

Comment:
I tried upgrading `@asyncapi/generator` to the latest version (2.0.0), but it still gives the following error:

```
Error: This template is not compatible with the current version of the generator (2.0.0). This template is compatible with the following version range: >=1.15.0 <2.0.0.
```

This is even after I converted our AsyncAPI specs to the latest version of AsyncAPI (v3.0.0) using [their recommended version converter](https://github.com/asyncapi/converter-js). I'm not sure why, but these are all "latest" versions so there may still be some kinks to work out.

So, instead, I pinned the generator to the latest version <2.0.0. We can unpin when the latest version is stabilized.